content/docs/gto/get-started.md

```diff
@@ -0,0 +1,170 @@
+a stage to a version. We can push to Git repository to start the CI, but let's
+ensure that changed our Registry first.
+
+```cli
+$ gto show
+╒══════════╤══════════╤═════════╤═════════╤════════════╕
+│ name     │ latest   │ #dev    │ #prod   │ #staging   │
+╞══════════╪══════════╪═════════╪═════════╪════════════╡
+│ churn    │ v3.1.1   │ v3.1.1  │ v3.0.0  │ v3.1.1     │
+│ segment  │ v0.4.1   │ v0.4.1  │ -       │ -          │
+│ cv-class │ v0.1.13  │ v0.1.13 │ -       │ -          │
+│ awesome  │ v0.0.1   │ -       │ -       │ -          │
+╘══════════╧══════════╧═════════╧═════════╧════════════╛
+```
+
+The `gto show` output confirms our expectation.
+
+## Acting downstream
+
+The power of using Git tags to register versions and assign stages is simple: we
+can act upon them in well-known way - in CI/CD.
+
+<details>
+
+### Click to set up a working repository to try this.
+
+Let's fork the [example-gto repo](https://github.com/iterative/example-gto) repo
+(you'll need a [GitHub](https://github.com/signup) account first). For CI/CD to
+start, you'll need to enable it on the "Actions" page of your fork.
+
+1. [Fork the repo](https://github.com/iterative/example-gto/fork). Make sure you
+   uncheck "Copy the `main` branch only" to copy Git tags as well:
+   <img width="877" alt="image" src="https://user-images.githubusercontent.com/6797716/199275275-439335f4-6f54-4cd7-910d-fc29ad3c095c.png">
+
+2. Enable workflows in your repo, for a Git tag to trigger CI:
+   <img width="869" alt="image" src="https://user-images.githubusercontent.com/6797716/199272682-dfd628bf-9599-4e85-a623-bf4a10c3d7e1.png">
+
+</details>
+
+Let's do the same thing we did locally, but for your remote repo. Don't forget
+to replace the URL:
+
+```cli
+$ gto assign cv-class --version v0.1.13 --stage dev \
+    --repo https://github.com/aguschin/example-gto
+Created git tag 'cv-class#dev#1' that assigns stage to version 'v0.1.13'
+Running `git push origin cv-class#dev#1`
+Successfully pushed git tag cv-class#dev#1 on remote.
+```
+
+Now the CI/CD should start, and you should see that we found out: it was
+`cv-class` artifact, version `v0.1.13` that was assigned to `dev` stage. Using
+this information, the step `Deploy (act on assigning a new stage)` was executed
+(while `Publish (act on registering a new version)` was skipped):
+
+<details>
+
+### CI/CD execution example
+
+<img width="875" alt="image" src="https://user-images.githubusercontent.com/6797716/199276636-bf996ad3-7d9c-4100-9f3c-6444730e4d19.png">
+
+If you want to see more CI examples, check out
+[the example-repo](https://github.com/iterative/example-gto/actions).
+
+</details>
+
+## Next steps
+
+Thanks for completing this Get Started!
+
+- If you want how to specify artifact's metainformation like `path`, `type` and
+  `description`, check out [User Guide](/doc/gto/user-guide/artifacts).
+- If you want to learn about using DVC to keep your artifact binaries in remote
```

Comment:
Minor but these seem more like misc. related links than next steps. What would be possible next steps actually? Creating registries? Other integrations?